components/prize_pool/wikis/arenaofvalor/prize_pool_legacy_custom.lua

```diff
@@ -0,0 +1,43 @@
+---
+-- @Liquipedia
+-- wiki=arenaofvalor
+-- page=Module:PrizePool/Legacy/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+
+local PrizePoolLegacy = Lua.import('Module:PrizePool/Legacy', {requireDevIfEnabled = true})
+
+local CustomLegacyPrizePool = {}
+
+-- Template entry point
+function CustomLegacyPrizePool.run()
+	return PrizePoolLegacy.run(CustomLegacyPrizePool)
+end
+
+function CustomLegacyPrizePool.customOpponent(opponentData, CACHED_DATA, slot, opponentIndex)
```

Comment:
if usdprize and localprize need adjustments, do points need adjustments too?

Reply:
as far I can remember, We've never run into similar case like **usdprize1 usdprize2** for points. So no, points need no adjustments.

Reply:
> if usdprize and localprize need adjustments, do points need adjustments too?

Looking at the old code, only usd & local had support for this. point2 & points3 were supported, but they were additional columns, not split